### ISSUE TITLE
Usage of validation set

### ISSUE THREAD

**Issue by iacopo97** [wimmerth/anyup#24]
I have seen your training pipeline I was wondering if there is a reason why you decided not to compute some similarity scores after each or some epochs on the validation set.

Thank you for your pateince

**Comment by wimmerth** [comment]
Hi, thanks for your interest in AnyUp!

That's a good question. I didn't observe any major spikes / instabilities during model training, hence I sticked to the training settings as in JAFAR, which was the starting point for this project.
